Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -930,22 +926,20 @@ protected Bitmap getMarkerBitmapFromView(String locationName, String imageName,
         }
         ImageView pointer = customMarkerView.findViewById(R.id.pointer_img);
         try {
-            if (imageName.equals("ny_ic_zone_pickup_marker")) {
-                pointer.setImageDrawable(context.getResources().getDrawable(context.getResources().getIdentifier("ny_ic_zone_pickup_marker_yellow", "drawable", context.getPackageName())));
+            pointer.setImageDrawable(context.getResources().getDrawable(context.getResources().getIdentifier(imageName, "drawable", context.getPackageName())));
+            if (markerType.equals(MarkerType.SPECIAL_ZONE_MARKER)) {
                 ViewGroup.LayoutParams layoutParams = pointer.getLayoutParams();
-                layoutParams.height = 40;
-                layoutParams.width = 40;
+                layoutParams.height = 45;
```

Comment:
these also or can we make this `wrap_content`?

Reply:
Then make it configurable from purs

Reply:
size of this icon will be fixed and in future also icon will be same so making it configurable might not be useful

Reply:
here size is changed because icon is changed